src/shipit_code_coverage/tests/test_report_generators.py

```diff
@@ -16,32 +16,27 @@ def test_zero_coverage(tmpdir,
         grcov_uncovered_function_artifact, jsvm_uncovered_function_artifact
     ], out_dir=tmp_path)
 
-    with open(os.path.join(tmp_path, 'zero_coverage_files.json'), 'r') as f:
-        zero_coverage_files = json.load(f)
-
-    assert set(zero_coverage_files) == set(['mozglue/build/dummy.cpp', 'toolkit/components/osfile/osfile-win.jsm'])
-
-    with open(os.path.join(tmp_path, 'zero_coverage_functions/mozglue_build_dummy.cpp.json'), 'r') as f:
+    with open(os.path.join(tmp_path, 'zero_coverage_reports/mozglue_build_dummy.cpp.json'), 'r') as f:
         assert set(json.load(f)) == set(['main'])
-    with open(os.path.join(tmp_path, 'zero_coverage_functions/js_src_jit_JIT.cpp.json'), 'r') as f:
+    with open(os.path.join(tmp_path, 'zero_coverage_reports/js_src_jit_JIT.cpp.json'), 'r') as f:
         assert set(json.load(f)) == set(['anUncoveredFunction'])
-    with open(os.path.join(tmp_path, 'zero_coverage_functions/toolkit_components_osfile_osfile.jsm.json'), 'r') as f:
+    with open(os.path.join(tmp_path, 'zero_coverage_reports/toolkit_components_osfile_osfile.jsm.json'), 'r') as f:
         assert set(json.load(f)) == set(['read', 'write'])
 
-    with open(os.path.join(tmp_path, 'zero_coverage_functions.json'), 'r') as f:
-        zero_coverage_functions = json.load(f)
+    with open(os.path.join(tmp_path, 'zero_coverage_report.json'), 'r') as f:
+        zero_coverage_reports = json.load(f)
 
-    expected_zero_coverage_functions = [
-        {'funcs': 1, 'name': 'mozglue/build/dummy.cpp'},
-        {'funcs': 2, 'name': 'toolkit/components/osfile/osfile.jsm'},
-        {'funcs': 1, 'name': 'js/src/jit/JIT.cpp'},
-        {'funcs': 1, 'name': 'toolkit/components/osfile/osfile-win.jsm'},
+    expected_zero_coverage_reports = [
+        {'funcs': 1, 'name': 'mozglue/build/dummy.cpp', 'uncovered': True},
+        {'funcs': 2, 'name': 'toolkit/components/osfile/osfile.jsm', 'uncovered': True},
+        {'funcs': 1, 'name': 'js/src/jit/JIT.cpp', 'uncovered': True},
+        {'funcs': 1, 'name': 'toolkit/components/osfile/osfile-win.jsm', 'uncovered': True},
```

Comment:
The only two fully uncovered files should be `'mozglue/build/dummy.cpp` and `toolkit/components/osfile/osfile-win.jsm`.